--- examples/mnist/convert_mnist_data.cpp	2015-10-20 00:08:12.341319752 -0600
+++ examples/mnist/convert_mnist_data.cpp	2015-10-20 00:08:17.457319667 -0600
@@ -86,7 +86,7 @@
     CHECK_EQ(mkdir(db_path, 0744), 0)
         << "mkdir " << db_path << "failed";
     CHECK_EQ(mdb_env_create(&mdb_env), MDB_SUCCESS) << "mdb_env_create failed";
-    CHECK_EQ(mdb_env_set_mapsize(mdb_env, 1099511627776), MDB_SUCCESS)  // 1TB
+    CHECK_EQ(mdb_env_set_mapsize(mdb_env, 536870912), MDB_SUCCESS)  // FIXED
         << "mdb_env_set_mapsize failed";
     CHECK_EQ(mdb_env_open(mdb_env, db_path, 0, 0664), MDB_SUCCESS)
         << "mdb_env_open failed";
